server/etcdserver/txn/txn.go

```diff
@@ -278,7 +282,7 @@ func Txn(ctx context.Context, lg *zap.Logger, rt *pb.TxnRequest, txnModeWriteWit
 	return txnResp, trace, err
 }
 
-func checkTxn(ctx context.Context, txnWrite mvcc.TxnWrite, rt *pb.TxnRequest, isWrite bool, lessor lease.Lessor, txnPath []bool) error {
+func checkTxn(ctx context.Context, txnWrite mvcc.TxnRead, rt *pb.TxnRequest, isWrite bool, lessor lease.Lessor, txnPath []bool) error {
```

Comment:
we should rename the argu as well. s/txnWrite/txnRead/

Reply:
+1

Reply:
Done